### PR TITLE
IOs Default To Using Intel Channel

### PR DESCRIPTION
**Pull request by Steelpoint**
# About the pull request

Intel Officers will now default to speaking into the intel squad channel instead of the Almayer general channel. 

# Explain why it's good for the game

Its best for the Intel squad to speak into the Intel comms. It may also slowly shift the IO culture towards cooperation and communication instead of solo ops. 

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
add: Intel Officers will now default to using the Intel radio channel when communicating. 
/:cl:

### CODE REVIEW

**Comment by morrowwolf** [comment]
Reference #4457 as it will have to be changed here as well once it is merged.

**Comment by Steelpoint** [comment]
Changes made

File: code/game/objects/items/devices/radio/encryptionkey.dm [hunk 1]
```diff
@@ -158,7 +158,7 @@
 /obj/item/device/encryptionkey/io
 	name = "\improper Marine Intelligence Officer Radio Encryption Key"
 	icon_state = "cap_key"
-	channels = list(RADIO_CHANNEL_COMMAND = TRUE, RADIO_CHANNEL_ENGI = TRUE, RADIO_CHANNEL_JTAC = TRUE, RADIO_CHANNEL_MEDSCI = TRUE, SQUAD_MARINE_1 = FALSE, SQUAD_MARINE_2 = FALSE, SQUAD_MARINE_3 = FALSE, SQUAD_MARINE_4 = FALSE, SQUAD_MARINE_5 = FALSE, SQUAD_MARINE_CRYO = FALSE, RADIO_CHANNEL_INTEL = TRUE)
+	channels = list(RADIO_CHANNEL_COMMAND = TRUE, RADIO_CHANNEL_ENGI = TRUE, RADIO_CHANNEL_JTAC = TRUE, RADIO_CHANNEL_MEDSCI = TRUE, SQUAD_MARINE_1 = FALSE, SQUAD_MARINE_2 = FALSE, SQUAD_MARINE_3 = FALSE, SQUAD_MARINE_4 = FALSE, SQUAD_MARINE_5 = FALSE, SQUAD_MARINE_CRYO = FALSE, RADIO_CHANNEL_INTEL = TRUE, RADIO_CHANNEL_ALMAYER = TRUE)
```
Comment (morrowwolf) [inline code/game/objects/items/devices/radio/encryptionkey.dm:161]: Remove intel from here and almayer channel should be the first in the list

**Review by morrowwolf — CHANGES_REQUESTED** [review]

**Comment by Steelpoint** [comment]
Requested changes made

**Pull request merged**